src/Components/ProposalCard/ProposalCardActionSummary.tsx

```diff
@@ -1,59 +1,25 @@
-import { Box } from 'Components/Primitives/Layout';
 import { Loading } from 'Components/Primitives/Loading';
-import { isDesktop } from 'react-device-detect';
-import { FiArrowRight } from 'react-icons/fi';
-import styled from 'styled-components';
+import { getInfoLineView } from 'old-components/Guilds/ActionsBuilder/SupportedActions';
+import UndecodableCallInfoLine from 'old-components/Guilds/ActionsBuilder/UndecodableCalls/UndecodableCallsInfoLine';
+import { useTypedParams } from 'Modules/Guilds/Hooks/useTypedParams';
+import { useFilteredProposalActions } from 'hooks/Guilds/guild/useFilteredProposalActions';
+import { Proposal } from 'Components/Types';
+import {
+  ActionsWrapper,
+  BorderedIconDetailWrapper,
+  NotFoundActionWrapper,
+} from 'Components/ProposalCard/ProposalCard.styled';
 
 interface ProposalCardActionSummaryProps {
-  isLoading?: boolean;
+  proposal?: Proposal;
 }
 
-const IconDetailWrapper = styled(Box)`
-  display: flex;
-  flex-direction: row;
-  align-items: center;
-  flex: 1;
-`;
-
-const BorderedIconDetailWrapper = styled(IconDetailWrapper)`
-  border: 1px solid ${({ theme }) => theme.colors.border.initial};
-  border-radius: 1rem;
-  padding: 0.25rem 0.8rem;
-  flex: none;
-  display: flex;
-`;
-
-const Detail = styled(Box)`
-  font-size: 0.95rem;
-  font-weight: 600;
-  margin-left: 0.5rem;
-`;
-
-const Icon = styled.img<{
-  spaceLeft?: boolean;
-  spaceRight?: boolean;
-  bordered: boolean;
-}>`
-  width: 1.5rem;
-  height: 1.5rem;
-  display: flex;
-  justify-content: center;
-  align-items: center;
-
-  ${props => props.spaceLeft && `margin-left: 0.5rem;`}
-  ${props => props.spaceRight && `margin-right: 0.5rem;`}
-
-  ${props =>
-    props.bordered &&
-    `
-    border: 1px solid #000;
-    border-radius: 50%;
-  `}
-`;
-
 const ProposalCardActionSummary: React.FC<ProposalCardActionSummaryProps> = ({
-  isLoading,
+  proposal,
 }) => {
+  const { guildId } = useTypedParams();
+  const actions = useFilteredProposalActions(guildId, proposal?.id);

```

Comment:
We should make this the same as our refactor structure so the component takes in the data as props and instead have a wrapper for this

Reply:
I include this in proposal wrapper  and pass it down to proposalcard. Is that ok?

Reply:
Yes that would be good for now, we might move to preferring to pass the component itself but for now yeah, pass actions as a prop via wrapper